localfs/readdir.go

```diff
@@ -0,0 +1,50 @@
+package localfs
+
+import (
+	"io"
+	"path"
+
+	"github.com/hugelgupf/p9/p9"
+)
+
+// Readdir implements p9.File.Readdir.
+func (l *Local) Readdir(offset uint64, count uint32) ([]p9.Dirent, error) {
+	var (
+		p9Ents = make([]p9.Dirent, 0)
+		cursor = uint64(0)
+	)
+
+	for len(p9Ents) < int(count) {
+		singleEnt, err := l.file.Readdirnames(1)
+		if err != nil {
+			if err == io.EOF {
+				err = nil // we want to return an empty RREADDIR, not an RERROR
+			}
```

Comment:
Not sure if we should be consuming this error or returning it.

Reply:
i think this should go something like this:

```go
if err == io.EOF {
  return p9Ents, nil
} else if err != nil {
  return nil, err
}
```

Reply:
Done: https://github.com/hugelgupf/p9/pull/12/commits/0ecadc9ec01c175c48af3d85aec6c93d07382bab